### ISSUE TITLE
Potential Double Application of Weight in loss_point Computation

### ISSUE THREAD

**Issue by johntr1** [ychensu/LRANet#8]
Hi. Thank you for the great work on LRANet! 

I am currently using your model as a baseline, and while reviewing the `forward_single` computation in `LRALoss`, I noticed what seems to be a double application of `weight` in the `loss_point` calculation:

```
loss_point = torch.sum(weight * F.smooth_l1_loss(
    p_gt[pos_idx],
    p_pre[pos_idx],
    reduction='none').mean(dim=-1))

loss_point = torch.sum(weight * loss_point)
```

The second line appears to essentially apply the weights again, which might distort the loss scale. I was wondering if this was intentional and if you would mind confirming if its purpose is to boost the loss or if it was possibly leftover from earlier experimentation?

Thanks again for sharing your code and research!

**Comment by ychensu** [comment]
Yes, it was leftover from earlier experimentation, and it has little effect on the final results. Thank you for pointing that out.
